exporters/metric/cortex/cortex.go

```diff
@@ -29,6 +46,13 @@ func (e *Exporter) ExportKindFor(*apimetric.Descriptor, aggregation.Kind) metric
 
 // Export forwards metrics to Cortex from the SDK
 func (e *Exporter) Export(_ context.Context, checkpointSet metric.CheckpointSet) error {
+	timeSeries, err := e.ConvertToTimeSeries(checkpointSet)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("%v", timeSeries)
```

Comment:
Looks like a debug log statement that should be removed or switched to log.debug?

Reply:
This is just to keep Go happy until timeseries gets used.